core/src/main/java/com/zextras/carbonio/files/config/interfaces/FilesConfig.java

```diff
@@ -0,0 +1,22 @@
+// SPDX-FileCopyrightText: 2024 Zextras <https://www.zextras.com>
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package com.zextras.carbonio.files.config.interfaces;
```

Comment:
Move the interface inf the config package, so that coherence with the rest of the codebase is preserved

```suggestion
package com.zextras.carbonio.files.config;
```